test/functional/ui/popupmenu_spec.lua

```diff
@@ -945,6 +945,82 @@ describe('ui/ext_popupmenu', function()
     }}
 
   end)
+
+  it('works with builtin popupmenu', function()
```

Comment:
```suggestion
  it('does not interfere with mousemodel=popup', function()
```

Reply:
Thanks for your comment.
We have corrected the wording.